readvault.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+if [ "$1" == "" ]  ||  [ "$2" == "" ];
+then
+	echo "USAGE: readvault.sh vaultkey propertypath"
+	echo "EXAMPLE: readvault.sh secret/pepper/dev/v1/conf .data.gcp.serviceKey.token_uri"
+	exit 1
+fi
+vault read -format=json $1 | jq -r $2
```

Comment:
Just trying to keep code a little more readable within config.yml . Maybe there are other chunks in the config.yml that can be refactored to external scripts?

Reply:
I think there's a convention of putting CI related scripts and files in a top-level directory called `ci`. You might want to consider that convention instead of a `build-utils` directory.

Reply:
Got a repo example? Would like to see what they are doing.
Was considering moving all the scripts and config files, like Dockerfile that have to do with building and deploying off the root into some other place. At first blush referring to all this sort of stuff as ci does not sound very descriptive of intent, but really open to ideas.

Reply:
Tried finding example, but they're not in the JavaScript world. Some that are seems to just put scripts in the `.circleci` directory: [angular](https://github.com/angular/angular/tree/master/.circleci), [react-native](https://github.com/facebook/react-native/tree/master/.circleci), [electron](https://github.com/electron/electron/tree/master/.circleci). Probably not representative -- couldn't find anything substantial since they either don't use Circle or their `config.yml` is simple.

Reply:
Good examples! I am inclined to leave things as they are for now, but certainly open to change things as we learn more and/or our needs expand.